package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "books-store",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@reduxjs/toolkit": "^1.9.5",
```

Comment:
- [ ] Great job for adding redux toolkit for state management. I think you equally need to add react-redux to effectively set up redux store in your app. I couldn't see this dependency. You can install it using `npm install react-redux` or `yarn add react-redux`.

Reply:
@vanina123  As the reviewer requested, please add react-redux to your project dependencies.